src/scripts/products.js

```diff
@@ -0,0 +1,114 @@
+(function () {
+var productsCard = document.querySelectorAll(".js-products");
+var jsBed = document.getElementsByClassName("bed-products");
+var jsChair = document.getElementsByClassName("chair-products");
+var jsSofa = document.getElementsByClassName("sofa-products");
+var jsTable = document.getElementsByClassName("table-products");
+var jsDining = document.getElementsByClassName("dining-products");
+var jsDoots = document.getElementById("js-dots");
+
+function cardChecker() {
+    if (productsCard[0].classList.contains("active")) {

```

Comment:
można by było uprościć tą funkcję, gdyż bardzo dużo kodu jest dokładnie takiego samego, a różnica jest tylko w 2 parametrach. Można by tutaj jakoś pokonfigurować (np. obiekt w JS albo atrybut data- w htmlu), użyć iteratora i zrobić to w mniejszej ilości linii kodu.